compiler/qsc_frontend/src/compile/tests.rs

```diff
@@ -478,12 +482,12 @@ fn package_dependency() {
         None,
     );
     let unit2 = compile(&store, &[package1], sources2, TargetProfile::Full);
+    assert!(unit2.errors.is_empty(), "{:#?}", unit2.errors);
 
-    let foo_id = LocalItemId::from(1);
```

Comment:
Is this change needed just to uncouple two blocks or LocalItemId::from(1) actually changes between two uses?

Reply:
Nope, the id is the same in both places. I was just removing it because I found it confusing more than helpful and it matches the pattern of the new test I added, but could revert if you think it's worth keeping. The ID does indeed stay the same between the two uses and could be a variable.